### DISCUSSION TITLE
Restore old functionality for History page

### DISCUSSION THREAD

**Discussion by ildar170975** [home-assistant/frontend#16505] [Dashboard]
Earlier before ~2022.11 we had a History page providing 2 use-cases:
1. (by default) Show graphs for predefined list of entities (chosen via yaml) - kind of "overview of important entities".
2. Show graph for one entity chosen via UI.

That predefined list of entities was configured in yaml.

Then this functionality was removed and replaced by a new one providing 1 use-case:
1. (nothing by default) Show graphs for one or several entities chosen via UI.

It was suggested by many users to restore the old functionality - but this seems to be ignored.
On 2023.5 yaml-settings were declared as deprecated & Docs were changed to completely remove that old functionality.

A possible way could be restoring the old functionality + using the current functionality:
1. (by default) Show graphs for predefined list of entities (chosen via yaml, as it was earlier).
2. Show graphs for one or several entities chosen via UI.

There were at least two Github issues:
https://github.com/home-assistant/frontend/issues/13792 (no feedback from Dev)
https://github.com/home-assistant/core/issues/74956 (closed recently)

I asked why this old functionality was removed, got no answer and was pointed to Discussion.
So - here is an issue in Discussions.

I would like know why the old functionality - working, documented, used by many users - was removed.
And I would suggest to implement the "enriched" functionality (old+new) described above.

**Comment by emufan** [comment]
BTW and related to this. There where a limitation on IOs/Webkit with the numer of entities shown. I remember limitation on canvas pixel size. This was there for years and not only my "all" history didn't show up there at all in these browsers.

Then it was factorized and the problem was gone 👍, because of a split every x entries.

But with the recent changes it is back again. So even if you would be able to display "all" or "lots of yaml defined ones" you would run again in the old /but now new) canvas problem.

This is even happening new when I select a lot of entities. => No display in these browsers again.
